generators/component/index.js

```diff
@@ -48,6 +48,17 @@ module.exports = generators.Base.extend({
         this.destinationPath(config.componentsDir + '/' + this.name + '/example.html'),
         _.extend(options, getComponentProps(this))
       );
+    },
+
+    // Generate a appropriate custom sass / less file for all the components
+    generateCustomStyleFile: function() {
```

Comment:
Rename this to something like: `generateAppStyleFile ` or `generateMainStyleFile `. Because this acts as the entry point @afamyial